README.md

```diff
@@ -145,6 +145,41 @@ $ make check-html
 
 Our knowledge about our different programs, our community, etc is managed via this repository, [CiviCRM](https://civicrm.org/), and documents stored in Google Drive. It is then added to the website as explained in our [Knowledge Management System documentation](https://we-are-ols.org/knowledge_management.html)
 
+## **Deploy Preview for Pull Requests**  
+
+This repository uses GitHub Actions to generate a **preview deployment** of pull requests. 
+This allows contributors to see changes live before merging them into the `main` branch.  
+
+### **How It Works**  
+1. When a pull request is created against `main`, the GitHub Actions workflow (`.github/workflows/deploy-preview.yml`) runs automatically.  
+2. The workflow:  
+   - Checks out the repository code.  
+   - Sets up Ruby and installs dependencies.  
+   - Builds the Jekyll site.  
+   - Deploys the built site to a separate repository (`ols-site-preview`) on the `gh-pages` branch.  
+   - Comments on the pull request with a preview link in the format:  
+     ```
+     🎉 A preview of this PR is available at: https://open-life-science.github.io/<PR_NUMBER>
```

Comment:
Little bit confused - are we deploying to this URL, or to `we-are-ols.org/ols-site-preview`?

Reply:
That's exactly right. 💯 

Sorry for the confusion! I'll update `deploy.yaml` and the `READme.md` with the right link (http://we-are-ols.org/ols-site-preview).

Reply:
Updated - thanks, Yo!